### PR TITLE
Issue #18277: Use HttpUtils and remove duplicate methods.

### LINKED ISSUE #18277: Update Servlet 4.0 tests to use `HttpUtils.findStringInReadyUrl` when possible
Use `HttpUtils.findStringInReadyUrl` when possible instead of rewriting very similar function in our test bucket.

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #18277

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletPathForDefaultMappingDefault.java [hunk 1]
```diff
@@ -92,10 +90,7 @@ public void testServletPathForDefaultMapping_Default() throws Exception {
...
     @Test
     public void testServletPathForDefaultMapping_Default_2() throws Exception {
-        String expectedResponse = "ServletPath = /index.html PathInfo = null";
-        String url = "http://" + server.getHostname() + ":" + server.getHttpDefaultPort() + "/" + APP_NAME + "/index.html";
-
-        testServletPathForDefaultMappingDefault(url, expectedResponse);
+        HttpUtils.findStringInReadyUrl(server, "/" + APP_NAME + "/index.html", "ServletPath = /index.html PathInfo = null");
     }
 
     private void testServletPathForDefaultMappingDefault(String url, String expectedResponse) throws Exception {
```
Comment (isaacrivriv) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletPathForDefaultMappingDefault.java]: Maybe this method can be removed? Not seeing it used anywhere with the changes

**Review by isaacrivriv — APPROVED** [review]
Looks good to me. Thanks!

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletPathForDefaultMappingDefault.java (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletPathForDefaultMappingDefault.java]: Thanks! I just removed it. I'll run another PB and merge if successful.

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#run-libby-bot

**Pull request merged**